documentation/index.html.md

```diff
@@ -55,9 +55,9 @@ Although in most cases you'll work with the managers directly, the services are
 
 **Epicenter.js**
 
-The Epicenter.js library is available from our tools: <a href="https://forio.com/tools/js-libs/1.7.1/epicenter.min.js" target="_blank">https://forio.com/tools/js-libs/1.7.1/epicenter.min.js</a>. To use it in your project, simply add
+The Epicenter.js library is available from our tools: <a href="https://forio.com/tools/js-libs/1.8/epicenter.min.js" target="_blank">https://forio.com/tools/js-libs/1.8/epicenter.min.js</a>. To use it in your project, simply add
```

Comment:
I believe this should be 1.8.0, not 1.8

(i.e., https://forio.com/tools/js-libs/1.7/epicenter.min.js is a 404, https://forio.com/tools/js-libs/1.7.0/epicenter.min.js exists)